pkg/cmd/roachtest/test_runner.go

```diff
@@ -1086,6 +1101,12 @@ func (r *testRunner) runTest(
 
 	var timedOut bool
 
+	if grafanaAvailable {
+		// This links to an auto-refreshing dashboard for this test. Since it uses a `now-15m` time to `now` range
+		// it will only show data while the test is running.
+		shout(ctx, l, stdout, "grafana metrics can be followed at: https://go.crdb.dev/roachtest-grafana-live/%s/%s note: this will only show data while the test is running",
```

Comment:
The note about "while the test is running" seems odd since in most cases it won't be running by the time it's read unless someone is tailing the log. Maybe it's worth removing it entirely or paraphrasing to something like "adjust time range to see past run"; even better, why not encode the test start time in the url?